dbms/tests/queries/0_stateless/01013_totals_without_aggregation.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. $CURDIR/../shell_config.sh
+
+EXCEPTION_SUCCESS_TEXT=ok
+
+# Must throw an exception
+EXCEPTION_TEXT="WITH TOTALS, ROLLUP or CUBE are not supported without aggregation"
+$CLICKHOUSE_CLIENT --query="SELECT 1 AS id, 'hello' AS s WITH TOTALS" 2>&1 \
```

Comment:
It's more convenient to write simple sql test and add special comment  after query `-- { serverError 49 }`, which means that client would expect exception with code 49.